File: src/main/java/com/market/ddanggyeo/user/controller/UserInfoController.java
package com.market.ddanggyeo.user.controller;

import com.market.ddanggyeo.user.dto.UserDto;
import com.market.ddanggyeo.user.service.UserInfoService;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(value = "/api/user")
public class UserInfoController {

    private final UserInfoService userInfoService;

    UserInfoController(UserInfoService userInfoService) {
        this.userInfoService = userInfoService;
    }

    public ResponseEntity<String> insertUserInfo(@RequestBody UserDto userDto) {
        return ResponseEntity.ok().body(userInfoService.insertUserInfo(userDto));
    }
}

File: src/main/kotlin/com/market/ddanggyeo/user/controller/UserInfoController.kt
package com.market.ddanggyeo.user.controller

import com.market.ddanggyeo.account.dto.AccountDto
import com.market.ddanggyeo.user.dto.UserDto
import com.market.ddanggyeo.user.service.UserInfoService
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/user")
class UserInfoController(private val userInfoService: UserInfoService) {

    @RequestMapping("/insert")
    fun insertUserInfo(@RequestBody userDto: UserDto) =
        ResponseEntity.ok().body(userInfoService.insertUserInfo(userDto))

}